trunner/target/base.py

```diff
@@ -82,6 +82,10 @@ def exec_dir(self) -> str:
         """Returns a directory where test binaries are located in rootfs."""
         return "/bin"
 
+    def root_dir(self) -> str:
+        """Returns a root directory."""
+        return f"{self.project_dir}/_fs/{self.name}/root"
```

Comment:
Then also other dirs in TargetBase should be changed.

```suggestion
    def root_dir(self) -> Path:
        """Returns a root directory."""
        return Path(f"{self.project_dir}/_fs/{self.name}/root")
```